source/client/process_impl.cc

```diff
@@ -385,7 +385,7 @@ bool ProcessImpl::run(OutputCollector& collector) {
   int i = 0;
   std::chrono::nanoseconds total_execution_duration = 0ns;
   for (auto& worker : workers_) {
-    auto sequencer_execution_duration = worker->sequencer().executionDuration();
+    auto sequencer_execution_duration = worker->phases().back()->sequencer().executionDuration();
```

Comment:
Can you explain why it's safe to pick just the last phase for this?  Why don't we need the duration of all of the phases?

Reply:
See above; this is safe now because of some implicit knowledge abo this first step to land phases internally; in a follow up this has to change